tests/test_receivers.py

```diff
@@ -165,3 +165,35 @@ def test_event_deletion(app, receiver):
         event.delete()
         assert event.response == {'status': 410, 'message': 'Gone.'}
         assert event.response_code == 410
+
+
+def test_event_status(app):
+    """Test event status reporting."""
+    data = json.dumps({'x': 40, 'y': 2})
+
+    class TestCeleryReceiver(CeleryReceiver):
+
+        def run(self, event):
+            """Change task state."""
+            from celery import current_task, states
+
+            event.response['message'] = event.payload['x'] + event.payload['y']
```

Comment:
This is what I am doing, but I still get the error. The exact receiver code is this:
```python
    @shared_task()
    def add(x, y):
        return x + y

    class TestCeleryReceiver(CeleryReceiver):
        def run(self, event):            
            result = add.apply_async((event.payload['x'], event.payload['y']))
            event.response['message'] = result.get()
```
Should I begin a new session somewhere?